db/seeds.rb

```diff
@@ -8,3 +8,4 @@
 require File.dirname(__FILE__) + '/seed_helper.rb'
 
 create_roles_and_permissions
+populate_languages
```

Comment:
The seed task already includes the task to fill in the languages. If the cap task executes each time deploy happens, but the first thing these task do is call `Role.delete_all` and `Language.delete_all`, but that won't rest the id's in the database will it? What would be the consequence of ActiveRecord associations for existing users with roles for example?